dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/helpers/JavaUtils.java

```diff
@@ -0,0 +1,125 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.cxf.helpers;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.apache.cxf.common.util.SystemPropertyAction;
+
+import com.ibm.ws.ffdc.annotation.FFDCIgnore;
+
+public final class JavaUtils {
+
+    /** Use this character as suffix */
+    static final char KEYWORD_PREFIX = '_';
+
+    /**
+     * These are java keywords as specified at the following URL.
+     * http://java.sun.com/docs/books/jls/third_edition/html/lexical.html#3.9
+     * Note that false, true, and null are not strictly keywords; they are
+     * literal values, but for the purposes of this array, they can be treated
+     * as literals.
+     */
+    private static final Set<String> KEYWORDS = new HashSet<>(Arrays.asList(
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Use of identifier that is a keyword in later versions of Java (NM_FUTURE_KEYWORD_USED_AS_IDENTIFIER)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-use-of-identifier-that-is-a-keyword-in-later-versions-of-java-nm-future-keyword-used-as-identifier).
The identifier is a word that is reserved as a keyword in later versions of Java, and your code will need to be changed in order to compile it in later versions of Java.